aviary/examples/run_reserve_mission_cruise_fixedtime.py

```diff
@@ -111,13 +34,12 @@
             "throttle_enforcement": "boundary_constraint",
             "fix_initial": False,
             "constrain_final": False,
-            "fix_duration": True,
+            "fix_duration": True,  # TBD remove in future
             "initial_bounds": ((149.5, 448.5), "min"),
-            "duration_bounds": ((60, 60), "min"),  # set this for fixed duration phases
+            "duration_bounds": ((45, 45), "min"),  # TBD remove in future
```

Comment:
should target_time and duration_bounds match?

Reply:
`duration_bounds` gets over-written in `check_and_preprocess_inputs()` for `target_time()` phases. We're forced to include something for duration_bounds right now because `check_phase_info()` requires it to be set otherwise it errors out.